internal/js_scanner/js_scanner.go

```diff
@@ -359,6 +359,11 @@ func NextImportStatement(source []byte, pos int) (int, ImportStatement) {
 				if next == js.MulToken {
 					currImport.ExportName = string(nextValue)
 				}
+
+				// if this is import.meta.*, ignore (watch for first dot)
+				if next == js.DotToken && len(specifier) == 0 {
```

Comment:
Tried to do a “safe” check here—basically if we encounter a '.' after “import” before we encounter anything else, don’t treat this as an import.